content/geoip/geolite2-free-geolocation-data.mdx

```diff
@@ -136,3 +136,17 @@ package the databases with your commercial products without the need for
 attribution. For more information, please [visit the GeoLite2 Commercial
 Redistribution
 page](https://www.maxmind.com/en/geolite2-commercial-redistribution).
+
+## Earn a commission with your GeoLite2 integration or application
```

Comment:
This seems like a lengthy heading compared to our other headings. Is there a way to shorten this?